docs/attributes-registry/aws.md

```diff
@@ -26,30 +26,30 @@ This document defines generic attributes for AWS services.
+| Attribute                                     | Type     | Description                                                                                | Examples                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Stability                                                        |
+| --------------------------------------------- | -------- | ------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `aws.dynamodb.attribute_definitions`          | string[] | The JSON-serialized value of each item in the `AttributeDefinitions` request field.        | `["{ \"AttributeName\": \"string\", \"AttributeType\": \"string\" }"]`                                                                                                                                                                                                                                                                                                                                                                                                                                                      | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `aws.dynamodb.attributes_to_get`              | string[] | The value of the `AttributesToGet` request parameter.                                      | `["lives", "id"]`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
hum do you know what is up with the trailing spaces here?

Reply:
You mean the table padding? This is added by the command `npm run fix:format` in the makefile for `attribute-registry-generation`. If I comment that out I get:
```
| `aws.dynamodb.attribute_definitions` | string[] | The JSON-serialized value of each item in the `AttributeDefinitions` request field. | `["{ \"AttributeName\": \"string\", \"AttributeType\": \"string\" }"]` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
| `aws.dynamodb.attributes_to_get` | string[] | The value of the `AttributesToGet` request parameter. | `["lives", "id"]` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```